core/domain/state_domain.py

```diff
@@ -797,7 +797,10 @@ class WrittenTranslations(python_utils.OBJECT):
     """
 
     def __init__(self, translations_mapping):
-        """Initializes a WrittenTranslations domain object."""
+        """Initializes a WrittenTranslations domain object.
+        Args:
```

Comment:
Add newline above this.

Reply:
I just did. Thanks

Reply:
The newline still isn't there.

Reply:
@edeediong

Reply:
I think I've corrected this now. Really sorry @kevinlee12

Reply:
Done